test/tests/test.xapiwrapper.js

```diff
@@ -0,0 +1,50 @@
+after(function () {
+  //nothing in here
+});
+
+describe('testing xapiwrapper.min.js', function () {
+
+  var ADL, onBrowser, should
+
+  before(function () {
+
+      onBrowser = false;
+      if (typeof window !== 'undefined') {
+          ADL = window.ADL;
+          onBrowser = true;
+      }
+      else {
+          ADL = require('../../dist/xapiwrapper.min');
```

Comment:
If the test is run in node, this is where the new umd wrapped xapiwrapper is loaded into node for testing.
I have taken the code from the `example.html`, swapped the request from synchronous to asynchronous and added as the first real test of the api in node. Other tests would need to follow.